src/moyo_heder.scss

```diff
@@ -0,0 +1,82 @@
+$main-bg-color: #00acdc;
+$header-background: white;
+$main-acent: #060b35;
+
+.header {
+  padding: 0 50px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  background: $header-background;
+  justify-content: space-between;
+}
+
+.nav {
+  margin-left: 0;
+  height: 100%;
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    margin-left: 20px;
+    white-space: nowrap;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14.06px;
+    height: 60px;
+    color: $main-acent;
+    display: flex;
+    align-items: center;
+  }
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo__img {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+}
+
+a.is-active.nav__link {
+  margin: 0;
+}
+
+.nav-list:first-child {
+  margin-left: 0;
+}
+
+.nav-hover:hover {
+  color: $main-bg-color;
+}
+
+.is-active {
+  position: relative;
+  color: $main-bg-color;
+}
+
+.nav-link:hover {
+  color: $main-bg-color;
+}
```

Comment:
The selector `.nav-link:hover` appears to be a typo or an incorrect BEM naming. It should match the element defined in lines 26-38, which would be `.nav__link:hover`.